include/GafferScene/Instancer.h

```diff
@@ -237,8 +237,8 @@ class GAFFERSCENE_API Instancer : public BranchCreator
 		struct PrototypeScope : public Gaffer::Context::EditableScope
 		{
 			PrototypeScope( const Gaffer::ObjectPlug *enginePlug, const Gaffer::Context *context, const ScenePath *parentPath, const ScenePath *branchPath );
-		private:
-			ScenePlug::ScenePath m_prototypePath;
+			private :
+				ScenePlug::ScenePath m_prototypePath;
```

Comment:
Oh, this is the same indentation as SceneScope, so maybe it was intentional, but I still want all member functions at the same indentation.